7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -117,44 +118,34 @@ get_lake_cell_xwalk <- function(lake_centroids, tile_cells) {
 # Map the query - grid cells, grid tiles, selected tile, cells w lakes in selected tile, lake centroids
 # For now, generating empty file for tiles that don't contain cells that contain lakes
 map_query <- function(out_file_template, lake_centroids, grid_tiles, grid_cells, cells_w_lakes) {
```

Comment:
Good point, but will probably be deleted. Only slightly confusing because it would get called from the targets pipeline which is only currently called from `7_drivers_munge.yml`